frontend/src/components/PopUp/index.js

```diff
@@ -32,20 +36,21 @@ export default function Pop({label, title, texts, listIndex, linkIndex}) {
           horizontal: 'center',
         }}
         onClose={handleClose}
-        // disableRestoreFocus
+        aria-live="polite"
       >
-        <div className="terms-wrapper" onClick={handleClose}>
-          {title}
-          
+        <div className="terms-wrapper" onClick={handleClose} >
+        {title}
+            <ul>
```

Comment:
Quick silly question, did you check that the `<a>` and the `<p>` cases don't look awful when wrapped by the `<ul>`? I'm pretty sure they shouldn't be _awful_... but just to be sure 😅

Reply:
I changed the logic around a bit to make it look less wonky (I'm a backend engineer...don't have much design specs). Let me know if the updated logic is okay? it's not very clean but it works!

Reply:
Sorry, I just read back what I wrote and I expressed myself terribly 🤦
- I didn't mean things _look_ bad, I meant things _might_ look bad and I was wondering if you'd checked (I hadn't!)
- I just checked and at this point in time things looked ok! 🎉
- The fix you introduced would actually change the way elements are rendered, causing ordering problems for whenever this component is fed a mix of paragraphs, links and list items

**Summarizing:** I'm sorry, let's revert your last commit and merge this thing!

Reply:
Oh -- I see thank you for explaining! Thanks again!!

Sorry about the confusion. I reverted it and think it's probably ready to go :)